src/engraving/dom/slurtie.cpp

```diff
@@ -164,6 +159,15 @@ void SlurTieSegment::spatiumChanged(double oldValue, double newValue)
     }
 }
 
+void SlurTieSegment::cloneLdata(SlurTieSegment* seg) const
+{
+    for (int i = 0; i < int(Grip::GRIPS); ++i) {
+        seg->m_ups[i]   = m_ups[i];
+        seg->m_ups[i].p = PointF();
```

Comment:
These offsets are not part of LayoutData, so I think they should remain in the copy constructor.